operator/controllers/csiscaleoperator_controller.go

```diff
@@ -468,7 +470,16 @@ func (r *CSIScaleOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		return ctrl.Result{}, err
 	}
 
-	csiNodeSyncer := clustersyncer.GetCSIDaemonsetSyncer(r.Client, r.Scheme, instance, daemonSetRestartedKey, daemonSetRestartedValue, CGPrefix)
+	cmData := map[string]string{}
+	cm, err := r.getConfigMap(instance, config.CSIEnvVarConfigMap)
+	if err != nil {
+		message := fmt.Sprintf("Failed to get the details of the configmap %s from cluster.", config.CSIEnvVarConfigMap)
+		logger.Error(err, message)
+		return ctrl.Result{}, nil
```

Comment:
If error is IsNotFound --> don't return here and return only in case of error other than IsNotFound (return ctrl.Result{}, err)

Reply:
Instead of `nil`, return `err` here, so that if there is any error apart from IsNotFound, it is returned back.